WEB_SERVICE/WEB/_HIGH/SSL_RACCOON_ATTACK/meta.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "Raccoon Attack on SSL/TLS",
+  "short_description": "A timing vulnerability in the Diffie-Hellman key exchange that allows attackers to recover sensitive data through precise measurement of server response times.",
+  "risk_rating": "high",
```

Comment:
high?

Reply:
yes the impact is high it can lead to session key recovery and decryption

Reply:
It should be low , different resources say so : https://nvd.nist.gov/vuln/detail/CVE-2020-1968 , https://security.paloaltonetworks.com/CVE-2020-1968